qiita_pet/test/test_study_handlers.py

```diff
@@ -532,8 +535,19 @@ def test_get_no_exist(self):
 
 
 class TestEBISubmitHandler(TestHandlerBase):
-    # TODO: add proper test for this once figure out how. Issue 567
-    pass
+    # TODO: add proper test for this once figure out how. Issue 567 (post)
```

Comment:
Took me a bit to realize that this comment is still valid, because it refers to the POST method, maybe think of rephrasing this.